<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>svTool</class>
 <widget class="QMainWindow" name="svTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>svTool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <property name="usesScrollButtons">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tabHome">
       <attribute name="title">
        <string>Home</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="home_rawInputGroup">
          <item row="1" column="0">
           <widget class="QPlainTextEdit" name="home_rawInputBox"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="home_rawInputLabel">
            <property name="text">
             <string>Input</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="home_statusLabel">
          <property name="text">
           <string>STATUS:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="home_typeGroup">
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="home_typeLabel">
            <property name="text">
             <string>Input Type</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QPushButton" name="home_validateButton">
            <property name="text">
             <string>Validate Input</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="home_keysLabel">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>Keys</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QSpinBox" name="home_keysSpinBox">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::UpDownArrows</enum>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>18</number>
            </property>
            <property name="value">
             <number>4</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabDebug">
       <attribute name="title">
        <string>Debug</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QGridLayout" name="debug_rawInputGroup">
          <item row="0" column="0">
           <widget class="QLabel" name="debug_rawInputLabel">
            <property name="text">
             <string>Raw Input</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QTextBrowser" name="debug_rawInputBox"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="debug_procInputGroup">
          <item row="0" column="0">
           <widget class="QLabel" name="debug_procInputLabel">
            <property name="text">
             <string>Processed Input</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QTextBrowser" name="debug_procInputBox"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabStutter">
       <attribute name="title">
        <string>Stutter</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0" rowspan="3">
         <layout class="QGridLayout" name="stutter_procInputGroup">
          <item row="0" column="0">
           <widget class="QLabel" name="stutter_procInputLabel">
            <property name="text">
             <string>Processed Input</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QTextBrowser" name="stutter_procInputBox"/>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <layout class="QGridLayout" name="stutter_initSVGroup">
          <item row="1" column="0" colspan="2">
           <widget class="QLabel" name="stutter_initSVLabel">
            <property name="text">
             <string>Initial SV</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="stutter_initSVSpinBox">
            <property name="decimals">
             <number>2</number>
            </property>
            <property name="minimum">
             <double>0.100000000000000</double>
            </property>
            <property name="maximum">
             <double>10.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>1.000000000000000</double>
            </property>
            <property name="value">
             <double>10.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QSlider" name="stutter_initSVSlider">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="pageStep">
             <number>1</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="2" rowspan="3">
         <layout class="QGridLayout" name="stutter_procOutputGroup">
          <item row="0" column="0">
           <widget class="QLabel" name="stutter_procOutputLabel">
            <property name="text">
             <string>Processed Output</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QTextBrowser" name="stutter_procOutputBox"/>
          </item>
         </layout>
        </item>
        <item row="1" column="1">
         <layout class="QGridLayout" name="stutter_thresholdGroup">
          <item row="1" column="0">
           <widget class="QSlider" name="stutter_thresholdSlider">
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="stutter_thresholdSpinBox">
            <property name="decimals">
             <number>0</number>
            </property>
            <property name="maximum">
             <double>100.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>1.000000000000000</double>
            </property>
            <property name="value">
             <double>50.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="stutter_thresholdLabel">
            <property name="text">
             <string>Threshold</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="1">
         <layout class="QGridLayout" name="stutter_averageSVGroup">
          <item row="0" column="0">
           <widget class="QLabel" name="stutter_averageSVLabel">
            <property name="text">
             <string>Average SV</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QDoubleSpinBox" name="stutter_averageSVSpinBox">
            <property name="minimum">
             <double>0.100000000000000</double>
            </property>
            <property name="maximum">
             <double>1.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0" colspan="3">
         <widget class="QPushButton" name="stutter_generateButton">
          <property name="text">
           <string>Generate</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>716</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
